core/src/test/java/org/apache/iceberg/rest/requests/TestCreateNamespaceRequest.java

```diff
@@ -73,76 +69,56 @@ public void testCanDeserializeWithoutDefaultValues() throws JsonProcessingExcept
   @Test
   public void testDeserializeInvalidRequest() {
     String jsonIncorrectTypeForNamespace = "{\"namespace\":\"accounting%00tax\",\"properties\":null}";
-    AssertHelpers.assertThrows(
```

Comment:
the reason I changed this to `Assertions.assertThatThrownBy` is because that I think the message in `AssertHelpers.assertThrows(..)` is rather noisy. Usually it's clear from looking at the code what's being expected and when the assertion itself fails, it will be descriptive enough to understand why the code fails.